PreciseAlloy.Frontend/src/pages/Contact.tsx

```diff
@@ -8,4 +9,15 @@ const Portfolio = () => {
   return <ContactTemplate header={header} footer={footer} contact={contact} alert={alert} />;
 };
 
-export default Portfolio;
+// export default Portfolio;
+
+export default {
```

Comment:
Declare should be automatic, we should read the Name file like it is a group, and variation like Name~Variation is an item. You don't want to declare this line on more files. I believe it. I think we can do it in the routes.ts

Reply:
I think Cuong's update for this makes sense, developer can custom the display name they want